src/resources/scripts/i18n/de/session.ts

```diff
@@ -33,7 +33,26 @@ const session = {
 		'Legen Sie die Datei hier ab, um sie hochzuladen.',
 	'dragAndDrop.explanation.outsideDropArea':
 		'Ziehen Sie die Datei in das Feld, um sie hochzuladen.',
-	'dragAndDrop.restrictions': '.jpg, .png, .pdf, .docx, .xlsx bis maximal 5MB'
+	'dragAndDrop.restrictions':
+		'.jpg, .png, .pdf, .docx, .xlsx bis maximal 5MB',
+	'encrypted.notice.send.info':
+		'Ich kann unsere Nachrichten aus technischen Gr&uuml;nden nicht mehr lesen.',
+	'encrypted.notice.send.first':
+		'Senden Sie die untenstehende Nachricht und benachrichtigen Sie ',
+	'encrypted.notice.send.second':
+		', dass Sie aus technischen Gründen keine Nachrichten lesen können.',
+	'encrypted.notice.first':
+		'Aus technischen Gründen können Sie keine Nachrichten lesen und schreiben. Wenn ',
+	'encrypted.notice.second':
+		' den Chat öffnet, können Sie die Nachrichten wieder lesen und neue Nachrichten schreiben.',
+	'encrypted.notice.more': ' Mehr erfahren',
+	'encrypted.overlay.headline': 'Ende-zu-Ende Verschlüsselung',
+	'encrypted.overlay.text.first':
+		'Ihre Nachrichten sind aus Sicherheitsgründen Ende-zu-Ende verschlüsselt. Das bedeutet, niemand außerhalb dieses Chats kann die Nachrichten lesen. Nicht einmal die Online-Beratungs-Plattform. <br/><br/>Da Sie Ihr Passwort zurückgesetzt haben, sind die Nachrichten für Sie momentan nicht lesbar. Sobald ',
+	'encrypted.overlay.text.second':
+		' den Chat öffnet, können Sie die Nachrichten wieder lesen und auch schreiben.',
+	'encrypted.overlay.button': 'Schliessen',
```

Comment:
```suggestion
	'encrypted.overlay.button': 'Schließen',
```
Bitte überall gleich ersetzten. Wir haben aktuell einen Mix aus Schliessen und Schließen